app/javascript/src/stores/editor.js

```diff
@@ -104,5 +104,7 @@ export const settings = writable({
   "color-invalid": "#b33834",
   "color-custom-keyword": "#c678dd",
   "show-indent-markers": true,
-  "word-wrap": false
+  "word-wrap": false,
+  "autocomplete-parameter-objects": false,
+  "autocomplete-min-parameter-size": 2
```

Comment:
PR description says 3, but this is set to 2. Which one should it be?

Reply:
Ah yes, it should be 2. I had it at 3 initially, but 2 felt better